Lib/test/test_ntpath.py

```diff
@@ -422,15 +422,20 @@ def test_ismount(self):
             self.assertTrue(ntpath.ismount(b"\\\\localhost\\c$"))
             self.assertTrue(ntpath.ismount(b"\\\\localhost\\c$\\"))
 
+    def assertEqualCI(self, s1, s2):
+        """Assert that two strings are equal ignoring case differences
+        """
+        self.assertEqual(s1.lower(), s2.lower())
```

Comment:
Should this not be str.casefold() instead?

Reply:
Unfortunately, this test checks unicode & bytes. Bytes doesn't support casefold. For the purpose, I'm going to assume that .lower will cover us until someone comes up with a case where it doesn't.